s3_scrubber/Cargo.toml

```diff
@@ -34,6 +34,4 @@ pageserver = {path="../pageserver"}
 tracing.workspace = true
 tracing-subscriber.workspace = true
 clap.workspace = true
-
-atty = "0.2"
 tracing-appender = "0.2"
```

Comment:
Could you fix this to a text file, add that missing newline in the end? I don't think I can suggest that.

your $EDITOR probably has a setting for producing only such files.

Reply:
Reformatted this file